config/api.yml

```diff
@@ -56,6 +56,20 @@
       :get:
       - :name: read
         :identifier: embedded_configuration_script_payload_view
+  :configuration_script_sources:
+    :description: Configuration Script Source
+    :options:
+    - :collection
+    :verbs: *g
+    :klass: ConfigurationScriptSource
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: embedded_configuration_script_source_view
```

Comment:
@bdunne There are two subclasses that UI can target ExternalAutomationManager and EmbededAutomationManager.  @lgalis and I discussed and felt that the External ones would be named without any decoration (e.g. configuration_script_source_view), but Embedded would be named with decoration (e.g. embedded_configuration_script_source_view)